package/cobalt/cobalt.mk

```diff
@@ -1,40 +1,62 @@
 COBALT_VERSION = 06cbe09f396bfbcac0330c336180150352af2a94
 COBALT_SITE = https://github.com/Metrological/cobalt
 COBALT_SITE_METHOD = git
+COBALT_INSTALL_STAGING = YES
+COBALT_DEPENDENCIES = host-bison alsa-lib
```

Comment:
Is alsa also needed in the Gstreamer case?

Reply:
Sorry I am not sure about this case also. This is already in the cobalt.mk, @srinivas-kakarla, can you please give your input on this ?

Reply:
@wouterlucas,
In the Cobalt browser, Starboard is the porting interface. We have replaced FFMPEG with GStreamer for audio decoding functionality. However Starboard uses ALSA for audio rendering.